radicle-httpd/src/api/v1/projects.rs

```diff
@@ -467,3 +467,402 @@ fn stats(repo: &Repository, head: Oid) -> Result<Stats, Error> {
+        ))
+        .await;
+
+        assert_eq!(
+            body,
+            json!({
+                "entries":[{
+                    "path":"dir1/README",
+                    "name":"README",
+                    "lastCommit":null,
+                    "kind":"blob"
+                }],
+                "lastCommit":null,
+                "name":"dir1",
+                "path":"dir1",
+                "stats":{
+                    "branches":1,
+                    "commits":2,
+                    "contributors":1
+                }
+            })
+        );
+    }
+
+    #[tokio::test]
+    async fn test_projects_remotes_root() {
+        let body = get_response(&format!(
+            "/v1/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/remotes"
+        ))
+        .await;
+
+        assert_eq!(
+            body,
+            json!([{
+                "id":"z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi",
+                "heads":{"master":"609769c057e54e63ef6e3220bb65118fb9bf0d5a"},
+                "delegate":false
+            }])
+        );
+    }
+
+    #[tokio::test]
+    async fn test_projects_remotes() {
+        let body = get_response(&format!(
+            "/v1/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/remotes/z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi"
+        ))
+        .await;
+
+        assert_eq!(
+            body,
+            json!({
+                "id":"z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi",
+                "heads":{"master":"609769c057e54e63ef6e3220bb65118fb9bf0d5a"},
+                "delegate":false
+            })
+        );
+    }
+
+    #[tokio::test]
+    async fn test_projects_blob() {
+        let body = get_response(&format!(
+            "/v1/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/blob/609769c057e54e63ef6e3220bb65118fb9bf0d5a/README"
+        ))
+        .await;
+
+        assert_eq!(
+            body,
+            json!({
+                "binary":false,
+                "content":"Hello World!\n",
+                "lastCommit":{
+                    "sha1":"ab58746ce58f605c757b613bb2056729e31750f9",
+                    "author":{
+                        "name":"Alice Liddell",
+                        "email":"alice@radicle.xyz"
+                    },
+                    "summary":"Initial commit",
+                    "description":"",
+                    "committer":{
+                        "name":"Alice Liddell",
+                        "email":"alice@radicle.xyz"
+                    },
+                    "committerTime":1673001014
+                },
+                "name":"README",
+                "path":"README"
+            })
+        );
+    }
+
+    #[tokio::test]
+    async fn test_projects_issues_root() {
+        let body = get_response(&format!(
+            "/v1/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/issues"
+        ))
+        .await;
+
+        assert_eq!(
+            body,
+            json!([{
+                  "id":body[0]["id"],
```

Comment:
`issues` are not stable (due to GIT_TIME), so they'll always have a different hash id, and timestamp.